deployment/kubernetes/k8s/overlays/example/kustomization.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+bases:
+- ../../base
+
+configMapGenerator:
+- name: sts-agent-config
+  behavior: merge
+  literals:
+     - STS_API_KEY=SOMEOTHER
```

Comment:
add `STS_STS_URL` `STS_PROCESS_AGENT_URL` `STS_APM_URL` to the overrides.
can we avoid specifying all of them and just use one variable (eg. `STS_URL`) ?

Reply:
@Voronenko `STS_PROCESS_AGENT_URL` `STS_APM_URL` are still missing in the agent-config overrides.
Did try to find a way to set the backend url only once?

Reply:
Reliable - no. There is concept of "variables"  (kustomization_explained.yaml in kustomization folder) - but does not work smoothly.

Ideally, if url issue could be resolved inside go agent itself. (Like prefer  STS_PROCESS_AGENT_URL , but use STS_STS_URL if not provided(